waveorder/cli/compute_transfer_function.py

```diff
@@ -61,18 +62,23 @@ def generate_and_save_phase_transfer_function(
     echo_headline("Generating phase transfer function with settings:")
     echo_settings(settings.phase.transfer_function)
 
+    settings_dict = settings.phase.transfer_function.dict()
     if settings.reconstruction_dimension == 2:
         # Convert zyx_shape and z_pixel_size into yx_shape and z_position_list
-        settings_dict = settings.phase.transfer_function.dict()
         settings_dict["yx_shape"] = [zyx_shape[1], zyx_shape[2]]
         settings_dict["z_position_list"] = list(
-            -(np.arange(zyx_shape[0]) - zyx_shape[0] // 2)
+            -(
```

Comment:
Can this be its own function with documentation?

Reply:
Great idea. Refactored.